x/evm/spec/08_params.md

```diff
@@ -77,3 +77,6 @@ By default, all block configuration fields but `ConstantinopleBlock`, are enable
 | MuirGlacierBlock    | 0                                                                    |
 | BerlinBlock         | 0                                                                    |
 | LondonBlock         | 0                                                                    |
+| ArrowGlacierBlock   | 0                                                                    |
+| GrayGlacierBlock    | 0                                                                    |
+| MergeNetsplitBlock  | 0                                                                    |
```

Comment:
Adding Shanghai and Cancun as well (see [chain_config.go](https://github.com/evmos/ethermint/blob/7f25e9bcc0/x/evm/types/chain_config.go#L50-L54))
```suggestion
| MergeNetsplitBlock  | 0                                                                    |
| ShanghaiBlock       | 0                                                                    |
| CancunBlock.        | 0                                                                    |
```